src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -165,13 +165,13 @@ public static function register($mode = 0)
 
                     exit(1);
                 }
-                if ('legacy' !== $group && !$isWeak) {
```

Comment:
the $isWeak variable is not used anymore, isn't it ?

Reply:
That's right, good riddance!

Reply:
but now $isVendor is always false and $inVendors is never called :)

Reply:
I spotted and pushed again yeah :P should have checked the tests before pushing 😅